bundles/org.openhab.binding.rotel/src/main/resources/OH-INF/thing/rcx1500.xml

```diff
@@ -8,6 +8,7 @@
 	<thing-type id="rcx1500">
 		<label>RCX-1500 Stereo Receiver</label>
 		<description>Connection to the Rotel RCX-1500 stereo receiver</description>
+		<semantic-equipment-tag>Receiver</semantic-equipment-tag>
```

Comment:
@andrewfg : can you explain what "Receiver" is for?
Is it for radio tuner?

Reply:
Receiver is the English (American) term for a typical audio visual amplifier e.g. https://europe.yamaha.com/en/products/audio_visual/av_receivers_amps/index.html -- strictly it describes an "amplifier-plus-other-features" which may include a radio tuner, but in fact includes other features like streaming, multi channel surround, dolby or other processing, video switching and pass through. In other words "receiver" has become a generic name for any a/v device. By contrast the term "amplifier" has come to mean only purist stereo amps without any other features.

Reply:
So you should have used Receiver more often for the Rotel devices (you largely used AudioVisual).
Is Receiver also adapted to a device that handles audio but not video ?

Reply:
AudioVisual is the parent of Receiver so if the thing is not self evidently the latter then it gets classified as the former.

Apropos the device that handles audio only, the same rule would apply. I.e. AudioVisual is essentially the catch all for home entertainment equipment where no more precise sub classification can clearly be applied.

Reply:
I'll add that in this list: https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv

`Receiver` has the synonyms: "Receivers, Audio Receiver, Audio Receivers, AV Receiver, AV Receivers"

So it should also cover at least some devices that handle only audio?

Reply:
When updating that file, maybe also think about adding a description that makes the result of this discussion clear to future users. The description will be shown as a tooltip in the UI.

Reply:
> think about adding a description

I will add that to the PR in core..